components/canvas/NodeTooltip.tsx

```diff
@@ -78,7 +78,7 @@ type Field<IncludeKey extends string, Key extends string> =
       [k in Key]: string | undefined;
     });
 
-type NodeTooltipProps = Pick<NodeTooltipContainerProps, "isEditing"> & {
+type NodeTooltipProps = Pick<NodeTooltipContainerProps, "isVisible"> & {
```

Comment:
I don't see isVisible being used as a prop in this component. This should be removed.

Reply:
Sorry, I have understood it differently now updated, since Pick creates a new type from existing type, I have removed it as it is already declared below.